### PR TITLE
node, npx: add Italian translation

### PR DESCRIPTION
**Pull request by ffranchina**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.it/common/node.md
Comment (kbdharun) [inline pages.it/common/node.md:24]:
```suggestion
`node --no-lazy --inspect-brk {{percorso/al/file}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. The translations LGTM, I have some suggestions for this page.

**Comment by ffranchina** [comment]
Thanks for your feedback! I am answering to each of them :)

File: pages.it/common/node.md
Comment (ffranchina) [reply pages.it/common/node.md:8]: In Italian is the preposition `to` could be rendered with `al` (preposition `a` + article `il`) in the case of the movement of something toward something else.
In the colloquial way, eventually, it is more widespread the usage of `del` (preposition `di` + article `il`) for this purpose (in English would be _path **of** file_).

The last option is also semantically more correct and accurate in Italian, in fact other files also make use of it (https://github.com/tldr-pages/tldr/blob/main/pages.it/common/more.md, https://github.com/tldr-pages/tldr/blob/main/pages.it/common/age.md, ..)

source: I am Italian

File: pages.it/common/node.md
Comment (ffranchina) [reply pages.it/common/node.md:24]: See answer above.

File: pages.it/common/node.md
Comment (ffranchina) [reply pages.it/common/node.md:20]: This is something that, especially in this kind of project where the double curly brackets have a specific meaning, can be very misleading since one could be tempted to translate it too while it has actually to do with the internals of `node`.

I opened a PR to fix this: https://github.com/tldr-pages/tldr/pull/8899

File: pages.it/common/node.md
Comment (mebeim) [inline pages.it/common/node.md:18]:
```suggestion
- Valuta un'espressione e stampa il risultato, utile per vedere le versioni delle dipendenze di node:
```

This translation is wrong. The original intent was to say that this can be used to return the result of an expression, the fact that it is being used to list the version of a package is coincidental.

File: pages.it/common/node.md
Comment (mebeim) [inline pages.it/common/node.md:22]:
```suggestion
- Attiva il debugger mettendo in pausa l'esecuzione finché il codice sorgente non viene caricato:
```

Simpler.

File: pages.it/common/node.md
Comment (mebeim) [reply pages.it/common/node.md:8]: @ffranchina we use `allo`/`al`/`alla` for paths in all Italian pages. It's better to follow the existing convention rather than mix and change it across pages. See for example: [cp](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/cp.md), [7z](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/7z.md), [chsh](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/chsh.md).

Comment (ffranchina) [reply pages.it/common/node.md:8]: It will give me shivers to do so but, if this is the policy we want to follow, so it be :)

File: pages.it/common/node.md
Comment (ffranchina) [reply pages.it/common/node.md:18]: Yeah, you're right. I did not know the `-p` flag and I tried to interpret the original translation. My bad, I will fix it.

File: pages.it/common/node.md
Comment (mebeim) [reply pages.it/common/node.md]: @ffranchina I see your point my friend :') I think for now it is better to follow the convention. A PR can always be open to discuss and change all occurrences from `al` to `del` in one go. Should be doable with some scripting and manual checks. If you decide to do that mention me in the PR.

**Review by mebeim — APPROVED** [review]
Thank you @ffranchina 🙂

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> This PR can be merged after #8899 is merged.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Blocked until #8899 is merged.

File: pages.it/common/node.md
Comment (ffranchina) [reply pages.it/common/node.md]: @mebeim Great to hear that, I'll propose such change then! :D

File: pages.it/common/npx.md
Comment (EmilySeville7cfg) [inline pages.it/common/npx.md:8]: Mandatory fix (no suggestion available): [ellipsis must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).
Warning: [no translation template for `{{arg1 arg2 ...}}` found](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).

File: pages.it/common/npx.md
Comment (EmilySeville7cfg) [inline pages.it/common/npx.md:16]: The same issue as above.

File: pages.it/common/npx.md
Comment (EmilySeville7cfg) [inline pages.it/common/npx.md:20]: The same issue as above.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! ❤️

**Comment by CleanMachine1** [comment]
Hi @ffranchina Could you sign the agreement so this can be merged.

**Comment by ffranchina** [comment]
Sorry if it took this much to sign it! :sweat:

**Pull request merged**
